.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,104 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const validScope = [
```

Comment:
Might want to add `build` here.

Maybe `raidemulator` as well, unless that should be grouped under `raidboss` in the future.

Reply:
Done. Thanks!

Reply:
Belated suggestion, but `i18n` as well? Mostly because I think marking translation patches help people looking through git log, and would help me making release notes (manually or automatically).

Reply:
done